src/Microsoft.DocAsCode.Metadata.ManagedReference/MergeComments/UidAndComment.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Metadata.ManagedReference
+{
+    internal sealed class UidAndComment
```

Comment:
I copied it from https://github.com/dotnet/docfx/blob/dev/tools/MergeDeveloperComments/Program.cs#L469 ...